### PR TITLE
 Fix failed e2e 

### PR DESCRIPTION
**Pull request by churik**
- added ETHERSCAN_API_KEY - it was the reason of errors "'<' not supported between instances of 'NoneType' and 'int'" and 'string indices must be integer'.
get_transactions and get_token_transactions returned message instead of expected array.

- added required password to custom mailserver

- added wallet.wait_balance_is_changed() to `test_backup_recovery_phrase_warning_from_wallet` - e2e tried to find element when balance still wasn't updated

- skipped some e2e with reasons (issues mostly)

- updated `test_navigation_history_kept_after_relogin`  and `test_back_forward_buttons_browsing_website`

- updated pytest.ini to avoid warnings

### CODE REVIEW

**Comment by churik** [comment]
test_can_see_all_transactions_in_history - Passed
https://ci.status.im/job/end-to-end-tests/job/status-app-prs/428/console

**Comment by churik** [comment]
In https://ci.status.im/job/end-to-end-tests/job/status-app-prs/430/:
Passed: 
`test_back_forward_buttons_browsing_website`
`test_can_see_all_transactions_in_history`
`test_send_eth_from_wallet_to_address`
`test_send_stt_from_wallet`
`test_fetching_balance_after_offline`

File: ci/tests/Jenkinsfile.e2e-nightly [hunk 1]
```diff
@@ -40,6 +40,10 @@ pipeline {
             usernameVariable: 'SAUCE_USERNAME',
             passwordVariable: 'SAUCE_ACCESS_KEY'
           ),
+          string(
+          credentialsId: 'etherscan-api-key',
+          variable: 'ETHERSCAN_API_KEY'
```
Comment (jakubgs) [inline ci/tests/Jenkinsfile.e2e-nightly]: Looks good, but indent the arguments.

Comment (churik) [reply ci/tests/Jenkinsfile.e2e-nightly]: ah OK, will do

**Comment by churik** [comment]
https://ci.status.im/job/end-to-end-tests/job/status-app-prs/431/console
Passed:
`test_add_switch_delete_custom_mailserver `
`test_navigation_history_kept_after_relogin `
`test_send_valid_amount_after_insufficient_funds_error `

**Review by jakubgs — APPROVED** [review]
LGTM!

File: test/appium/tests/atomic/dapps_and_browsing/test_deep_links.py [hunk 2]
```diff
@@ -39,6 +41,7 @@ def test_open_user_profile_using_deep_link(self):
 
     @marks.testrail_id(5442)
     @marks.medium
+    # TODO: uncomment after https://github.com/status-im/universal-links-handler/pull/29 will be implemented in app
```
Comment (Serhy) [inline test/appium/tests/atomic/dapps_and_browsing/test_deep_links.py:45]: I guess you forgot 'marks.skip' here

**Review by Serhy — APPROVED** [review]

**Pull request merged**